Documentation/ViewDesigner/CommonSteps.md

```diff
@@ -62,6 +63,19 @@ Within your query add this line: _| where TimeGenerated {TimeRange}_
 
 **Updated:** _search * **| where TimeGenerated {TimeRange}** | summarize count() by Type_
 
+## Auto-Refresh Workbooks <a id="AutoRefresh"></a>

```

Comment:
Just checking, is auto refresh supposed to be in the ViewDesigner docs?

Reply:
I couldn't find any other location where I should put it. I didn't see any gallery related docs. Do you have some other location in mind where I can put this?

Reply:
possibly add a new section to the root level readme.md with a link to a new doc that is an overview where the links to datasources and visualizations are?  i'm not sure where this public version comes from: https://docs.microsoft.com/en-us/azure/azure-monitor/visualize/workbooks-overview, it might be a combination of several other docs?

also, you don't need the html `a` tag in the content, the markdown will generate things like that itself

Reply:
Done. Moved to a new doc